### DISCUSSION TITLE
Check Out not Functioning

### DISCUSSION THREAD

**Discussion by brianschroeder** [LeonShams/BestBuyBulletBot#9] [General]
Hi Leon!

Firstly, just wanted to say great work on the bot. The code and functionality looks great.

I was wondering if you had any tips for getting the checkout to work.

I am using firefox, and have my credit card info saved and my shipping address.

It seems the bot is having an issue dropping the ccv number once it gets to the checkout page.

Let me know if there's a place the log is stored so I can investigate.

Thanks,
Brian!

**Comment by LeonShams** [comment]
Thank you for the kind words, but regarding your problem, the bot itself doesn't store any logs, but I can investigate the issue and push and update to fix the bug if you could provide me with the steps taken by the bot so I can replicate the issue on my end.

Something like the following describing what happened would really help me out.

1. Set the bot to track the URL: https://www.bestbuy.com/site/apple-airpods-pro-white/5706659.p?skuId=5706659
2. During auto-checkout it opened up a browser window at the cart.
3. It clicked "proceed to checkout"
4. The input field for the CVV was then present but the bot just hung not doing anything.

**Reply by brianschroeder** [reply]
Sure thing, thanks for getting back to me.

1. Set the Bot to track https://www.bestbuy.com/site/pny
-elite-turbo-attache-4-32gb-usb-3-0-type-a-flash-drive-black-gray/1498843.p?skuId=1498843&intl=nosplash (For Testing)
2. The Bot was able to add the Item to the cart and get to the Review & Place Your Order Checkout Page.
3. Once on the Review & Place Your Order, the CVV field remains empty and the BOT hangs and does not take further action.


Also, please let me know if I should turn this into an issue.

Thanks,
Brian

**Reply by LeonShams** [reply]
Thanks a lot! The code should now be updated to enter the CVV on the "Review & Place Your Order" checkout pages. To update your current installation just run the same command you used to install the bot found in the readme.

**Comment by brianschroeder** [comment]
Thanks for the work above! Looks like something else came up will open an issue.
